### DISCUSSION TITLE
Class Attribute already exists and doesn't extend CI_Model Error

### DISCUSSION THREAD

**Discussion by Jym-ship-it** [opensourcepos/opensourcepos#4047] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/a48417a7-bf9b-478a-9797-8bcc12abb33a)
Can Some one help me with this ? so i just uploaded it in free hosting website , but i get this error , when i tried it in localhost it is running

**Comment by jekkos** [comment]
My guess is that you are using php8 with 3.3.9. Its. Of supported yet.

**Comment by jekkos** [comment]
You need to use php7.4 with the official release.

**Comment by Jym-ship-it** [comment]
But it said in install.md upto php 8

**Reply by jekkos** [reply]
Ah this is only valid for current master version which is not released yet, sorry about that

**Comment by jekkos** [comment, marked as answer]
I have clarified this in the INSTALL.md now.
